packages/zwave-js/src/lib/node/Node.ts

```diff
@@ -948,10 +948,19 @@ export class ZWaveNode extends ZWaveNodeMixins implements QuerySecurityClasses {
 				`skipping interview because it is already completed`,
 			);
 			return true;
-		} else {
-			this.driver.controllerLog.interviewStart(this);
 		}
 
+		if (this.skipInterview) {
+			this.driver.controllerLog.logNode(
+				this.id,
+				"node cannot be inteviewed",
+				"warn",
+			);
+			return true;
+		}
```

Comment:
I think it would make more sense to move this check to the start of `Driver.interviewNodeInternal(...)`